docs/docsite/rst/conf.py

```diff
@@ -283,3 +283,10 @@
                        'ansible_2_6': ('https://docs.ansible.com/ansible/2.6/', (None, '../ansible_2_6.inv')),
                        'ansible_2_5': ('https://docs.ansible.com/ansible/2.5/', (None, '../ansible_2_5.inv')),
                        }
+
+# linckchecker settings
+linkcheck_ignore = [
+    r'http://irc\.freenode\.net',
+]
+
```

Comment:
Adding workers may make it faster:
```suggestion
linkcheck_workers = 25
```


Also FTR this feature is known to be flaky because there's no robust retries: https://github.com/sphinx-doc/sphinx/issues/7388.